settings.gradle.kts

```diff
@@ -0,0 +1,18 @@
+@file:Suppress("UnstableApiUsage")
```

Comment:
Can we add a short comment which API requires us to use this `Suppress` statement?

Reply:
I can do that. These commands require the annotation:

![Bildschirmfoto_20250226_232523](https://github.com/user-attachments/assets/6b3b5e61-16a0-48aa-9601-a8cc9404aab9)

They all have a warning similar to:

"'getRepositoriesMode()' is marked unstable with @Incubating"